src/LocalRegistry.jl

```diff
@@ -555,4 +576,50 @@ function commit_registry(pkg::Pkg.Types.Project, package_path, package_repo, tre
     run(`$git commit -qm $message`)
 end
 
+function commit_title(pkg, new_package)
+    registration_type = new_package ? "New package" : "New version"
+    return "$(registration_type): $(pkg.name) v$(pkg.version)"
+end
+
+# Construct the git push options that will automatically create a PR
+# if the registry repo is hosted on GitLab.
+function gitlab(branch, pkg, new_package, repo, commit)
+    # If `branch` hasn't been set, create a branch name.
+    if isnothing(branch)
+        branch = string(pkg.name, "/v", pkg.version)
+    end
+
+    title = commit_title(pkg, new_package)
+    description = """
+    * Registering package: $(pkg.name)
+    * Repository: $(repo)
+    * Version: v$(pkg.version)
+    * Commit: $(commit)
```

Comment:
One thing which Registrator.jl does is tag the GitLab user that triggers the registration, which is quite useful because GitLab will notify that person as the PR progresses (notably if RegistryCI tests fail and the PR needs attending).

Could we add an optional argument to this method (and parent methods) in order to add e.g.

```
* Created by: @$(user_handle)
```
?

Reply:
Oh, I had forgotten to refresh my tab for a while and didn't see the PR got merged 😅 

Do you think this would be ok to add? Happy to make a PR since it should be quite straightforward.

Reply:
It seems slightly much to have an additional argument (to `register`) for this but I can see the usefulness, so please go ahead.

Reply:
Maybe it would be better to just pick up the author from some standard GitLab CI variable?

Reply:
Good idea, thanks!

Created https://github.com/GunnarFarneback/LocalRegistry.jl/pull/51